src/puppet-web/event.ts

```diff
@@ -244,6 +244,10 @@ async function onServerLogin(this: PuppetWeb, data, attempt = 0): Promise<void>
   // if `login` event fired before this.bridge inited, we delay the event for 1 second.
```

Comment:
`const args = Array.prototype.slice.call(arguments)`

Reply:
OK, has used args to replace argments.